VoiceTest/package.json

```diff
@@ -1,24 +1,27 @@
 {
   "name": "VoiceTest",
   "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "android": "react-native run-android",
+    "ios": "react-native run-ios",
+    "start": "react-native start",
+    "test": "jest"
+  },
   "dependencies": {
-    "react": "16.6.0-alpha.8af6728",
-    "react-native": "0.57.4",
-    "react-native-voice": "^0.2.6"
+    "react": "16.9.0",
+    "react-native": "0.61.5",
+    "react-native-voice": "file:../"
   },
   "devDependencies": {
-    "babel-jest": "23.6.0",
-    "jest": "23.6.0",
-    "metro-react-native-babel-preset": "0.49.0",
-    "react-test-renderer": "16.6.0-alpha.8af6728"
+    "@babel/core": "^7.6.2",
+    "@babel/runtime": "^7.6.2",
+    "babel-jest": "^24.9.0",
+    "jest": "^24.9.0",
+    "metro-react-native-babel-preset": "^0.56.0",
+    "react-test-renderer": "16.9.0"
```

Comment:
package.json should be shared with the root, so we can easily run the example.

Reply:
I'm trying to get the example to work with a single `package.json` from the root, but i'm unable to run them. :thinking:
running `npx react-native start` will look for an `index.js` in the root directory, not in the `example` folder. is there a way to make them work with a single package.json in the root?

Reply:
never mind, i saw the examples in `react-native-webview` which uses --root option for the cli. let me try with that :+1:

Reply:
I've tried it with the way `react-native-webview` does it, but `react-native start` stills tries to find an `index.js` in the root directory. it works when I add an `index.js` in the root directory, but I'm not sure if that is what we want (we don't really want to ditribute an extra content that is only used in the `example` folder)
I'm gonna give it another try tomorrow!

Reply:
Hmm maybe we can move the index file from `/src` to the root? There is nothing else in the folder. What do you think about that?

Reply:
@Naturalclar

Reply:
Just chipping in here, do you really need to use the same `package.json`? You can keep them separate and have a script on the root-level one to run the example like `react-navigation` does: https://github.com/react-navigation/react-navigation/blob/master/package.json#L23

Reply:
@safaiyeh I agree with @lucasbento that examples can have its own `package.json`.
I think the example folder should serve as a reference for users that wants to use the package, and having a `package.json` to know how its structured might be helpful to the users. Also, we can keep unrelated devDependencies just for examples out of the root `package.json`.
I've made some scripts that can run examples for the root `package.json`, like how its done in react-navigation. How do you feel? (That and I'm having a little hard time getting the single package.json to work :sob:)

Reply:
Okay no worries since it has been a pain! @Naturalclar approved ✅